### DISCUSSION TITLE
Just a message to say thanks!

### DISCUSSION THREAD

**Discussion by LyAhn** [smartavionics/Cura#99] [General]
Hey Mark,

I just want to say thanks for your hard work with your version of Cura. I was intially searching for the monotonic feature for Cura ever since it was introduced into PrusaSlicer and after a few days of hard searching I pretty much gave up. **Then** after searching for a fix for an entirely different issue, I noticed a comment of yours on the Ultimaker forums about your implementation of it just by chance. Strange how when you stop looking for something it just jumps out at you randomly.

I honestly love the work you've put into your version and really do appreciate the amount of effort you've gone through. I'm very much looking forward to seeing what you add in the future, maybe a possibility of giving us more options in terms of how monotonic fill is applied (letting us choose from Line or ZigZag patterns for example). So, thanks again Mark! 
You've made my printing life so much easier without the need of jumping from PS to Cura depending on what I'm printing.

**Comment by smartavionics** [comment]
Hi LyAhn, I'm glad you are happy with my releases. As for monotonic zigzag infill, I would implement that if I knew how to do it but at the moment I haven't come up with a scheme. Does PS provide monotonic zigzag infill or just lines like I have done?

**Reply by LyAhn** [reply]
Hey Mark, I think the line infill works all the same but I could probably see a tiny speed increase and nicer surface quality with zigzagging though by how much is questionable.

And looking a bit deeper into how PS works, I think it's just how they've done the gcode pathing? Unless I'm overlooking something in the settings, it only generates zigzag paths whereas Cura gives you more options.
Prusa:
![Image of Prusa pathing](https://jezz.wtf/img/psMonoPathing.png)
Cura:
![Image of Cura pathing](https://jezz.wtf/img/curaMonoPathing.png)

**Comment by bearfilm** [comment]
I second the comment - THANK YOU for taking time to make this happen!!
